packages/playground-ui/src/components/assistant-ui/attachments/voice-adapter.ts

```diff
@@ -0,0 +1,71 @@
+import { SpeechSynthesisAdapter } from '@assistant-ui/react';
+import { Agent } from '@mastra/core';
+
+export class VoiceAttachmentAdapter implements SpeechSynthesisAdapter {
```

Comment:
ℹ️ this is how we implement attachements in assistant-ui =>https://www.assistant-ui.com/docs/guides/Speech